core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -296,7 +296,7 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
     }
 
     //all workflow records in this status response list
-    val workflowsWithStatuses = response.statusResponse.collect {
+    val allWorkflows = response.statusResponse.collect {
```

Comment:
You renamed the variable, but am I correct that this is still marking the same number of workflows as having failed as it did before?

Reply:
Yep, that's the intention!